plugins/task-resources/src/components/state/StatesEditor.svelte

```diff
@@ -83,8 +95,76 @@
   async function onAdd (_class: Ref<Class<State | DoneState>>) {
     dispatch('add', _class)
   }
+
+  async function onDescriptionChange (value: string) {
+    dispatch('descriptionChange', { value })
+  }
+
+  async function onShortDescriptionChange (value: string) {
+    dispatch('shortDescriptionChange', { value })
+  }
 </script>
 
+{#if template?.space === 'recruit:space:VacancyTemplates'}
+  <div class="flex-no-shrink flex-between trans-title uppercase">
+    <Label label={task.string.ShortDescription} />
+  </div>
+  <div class="mt-3">
+    <EditBox
+      kind={'small-style'}
+      value={template?.shortDescription ?? ''}
+      on:change={() => onShortDescriptionChange(template?.shortDescription ?? '')}
+    />
+  </div>
+  <div class="mt-9">
+    <div class="flex-no-shrink flex-between trans-title uppercase">
+      <Label label={task.string.Description} />
+    </div>
+    <div class="mt-3">
+      {#key template?._id}
+        <StyledTextBox
+          emphasized
+          alwaysEdit
+          showButtons={false}
+          content={template?.description ?? ''}
+          on:value={(evt) => onDescriptionChange(evt.detail)}
+        />
+      {/key}
+    </div>
+  </div>
+  <div class="antiSection mt-9 mb-9">
+    <div class="antiSection-header">
+      <div class="antiSection-header__icon">
+        <Icon icon={tracker.icon.Issue} size={'small'} />
```

Comment:
Better move this icon to task and remove tracker dependency

Reply:
So what about components from tracker `tracker.component.CreateIssue` and `tracker.component.RelatedIssues`? I use them in this module too